krita_plugin/krita_diff/krita_diff.py

```diff
@@ -149,6 +151,7 @@ def img2img(self, path, mask_path, mode):
             "max_size": self.cfg('img2img_max_size', int),
             "seed": self.cfg('img2img_seed', str) if not self.cfg('img2img_seed', str).isspace() else '',
             "tiling": tiling,
+            "invert_mask": self.cfg('img2img_invert_mask', bool),
```

Comment:
This option is called `mask_invert` in krita_server.py. So it doesn't really work.